src/App.tsx

```diff
@@ -14,6 +14,9 @@ import Video from './components/Video/Video';
 import TroubleShootCard from './components/Troubleshoot/TroubleShootCard';
 import { ThemeProvider } from '@material-ui/core/styles';
 import theme from './CustomStyles';
+import { useSelector } from 'react-redux'; 
+import { RootState } from './store/reducers';
```

Comment:
Ooh, would it be possible to remove the useSelector and RootState imports since we aren't using them in the App.tsx file? :0

Reply:
yes!